### PR TITLE
Splitting module 4 in two parts to ease readability.

### PR DESCRIPTION
**Pull request by gkunz**

### CODE REVIEW

File: Contributions-4-1-Ind.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+---
+layout: default
+title: Open Source Contributions
+nav_order: 2
+has_children: true
+permalink: /Contributions-4-1-Ind
+---
```
Comment (YasinAhmed2) [inline Contributions-4-1-Ind.md:7]: Looks great with a straight forward and concise title: Open Source Contributions.

File: Contributions-4-2-Ind.md [hunk 2]
```diff
@@ -1,62 +1,11 @@
 ---
 layout: default
-title: Engaging in a FOSS project as a company employee
+title: Open Source Contributions
```
Comment (YasinAhmed2) [inline Contributions-4-2-Ind.md:3]: The Contributions-4-1-Ind.md and Contributions-4-2-Ind.md , seems to have the same title "Open Source Contributions".  I am not good for Markdown chapters. What is the intention at the file level? Could we keep here the subchapter title only?

**Review by YasinAhmed2 — COMMENTED** [review]
Great suggestion to split Chapter 4.  I have put some comments and a question on titling for Contributions-4-2-Ind.md. Otherwise, I approve the changes.

File: Contributions-4-2-Ind.md (hunk 2 above)
Comment (gkunz) [reply Contributions-4-2-Ind.md:3]: Good question. This entire front matter (between the dashes) is only used by the markdown rendering engine and the resulting theme. In our case, the title attribute is not used (and rendered) at all by the selected theme.

Take a look at the resulting GitHub page: https://expertlearninglab.github.io/foss-learning/Contributions-4-Ind (change the URL to check the other sections).

So, we could either remove the title or just keep a consistent naming. I renamed the title here to make it consistent with the other sections / files.

Comment (YasinAhmed2) [reply Contributions-4-2-Ind.md:3]: Let us keep it for a consistent naming

**Review by YasinAhmed2 — APPROVED** [review]
Chapter 4 is split in two subchapters  to ease readability

**Pull request merged**